setup.py

```diff
@@ -15,7 +15,7 @@
     packages=setuptools.find_packages(),
     entry_points={
         'console_scripts': [
-            'gitdir = gitdir.__main__:main',
+            'gitdir = gitdir.gitdir:main',
```

Comment:
The reason why the old way downloads twice:

console script would find __ main __.py::main() and execute it.

But there's yet another main() call is inside __ main __.py, for package access (python -m gitdir)